src/ResourceManagement/Scheduler/Microsoft.Azure.Management.Scheduler/Generated/Models/ServiceBusBrokeredMessageProperties.cs

```diff
@@ -21,7 +21,7 @@ public partial class ServiceBusBrokeredMessageProperties
         /// Initializes a new instance of the
         /// ServiceBusBrokeredMessageProperties class.
         /// </summary>
-        public ServiceBusBrokeredMessageProperties(string contentType = default(string), string correlationId = default(string), bool? forcePersistence = default(bool?), string label = default(string), string messageId = default(string), string partitionKey = default(string), string replyTo = default(string), string replyToSessionId = default(string), DateTime? scheduledEnqueueTimeUtc = default(DateTime?), string sessionId = default(string), DateTime? timeToLive = default(DateTime?), string to = default(string), string viaPartitionKey = default(string))
+        public ServiceBusBrokeredMessageProperties(string contentType = default(string), string correlationId = default(string), bool? forcePersistence = default(bool?), string label = default(string), string messageId = default(string), string partitionKey = default(string), string replyTo = default(string), string replyToSessionId = default(string), DateTime? scheduledEnqueueTimeUtc = default(DateTime?), string sessionId = default(string), TimeSpan? timeToLive = default(TimeSpan?), string to = default(string), string viaPartitionKey = default(string))
```

Comment:
@pinwang81 this looks like a breaking change as the datatype has changed.
You would have to bump up the major version.